geometry-1/DOMMatrix-002.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Geometry Interfaces: DOMMatrixReadOnly methods do not mutate the object</title>
+    <link rel="help" href="https://www.w3.org/TR/geometry-1/#dom-dommatrixreadonly">>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <p>Test DOMMatrixReadOnly methods do not mutate the object</p>
+    <div id="log"></div>
+    <script>
+
+        function initialMatrix(){
+          var matrix = new DOMMatrixReadOnly();
+          var initial = {m11:1,   m12:-0.5, m13: 0.5,  m14:0,
+                         m21:0.5, m22:2,    m23: -0.5, m24:0,
+                         m31:0,   m32:0,    m33: 1,    m34:0,
+                         m41:0,   m42:0,    m43: 0,    m44:1,
+                         is2D: false,
+                         isIdentity:false };
+          for(var p in initial) {
+            matrix[p] = initial[p];
```

Comment:
This doesn't do anything. These members are readonly. Maybe you want `var matrix = DOMMatrixReadOnly.fromMatrix(initial)`? (without `isIdentity` in initial, since that's not part of the spec's dictionary)